pkg/controllers/nodetemplate/suite_test.go

```diff
@@ -362,4 +365,64 @@ var _ = Describe("AWSNodeTemplateController", func() {
 			Expect(nodeTemplate.Status.SecurityGroups).To(BeNil())
 		})
 	})
+	Context("AMI Status", func() {
+		BeforeEach(func() {
+			awsEnv.EC2API.DescribeImagesOutput.Set(&ec2.DescribeImagesOutput{
```

Comment:
These have the same instance type requirements, right? Doesn't this mean that only one of these will be chosen? And since they all have the same creation time, won't it be random which one is chosen?

Reply:
For the status section we would expect all of them to show up as they all have the possibility of being chosen. The amiSelector that is being used is `{"*": "*"}`, we should have all the amis values in the node template.